src/migrations/20220514235230_game-messages-varchar-to-text.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async function (knex) {
+    await knex.schema.alterTable("game_messages", function (table) {
+        table.increments();
+        table.dropUnique(
+            ["category", "title", "message"],
+            "end_game_messages_category_title_message_unique"
+        );
+    });
+
+    // for some reason, cant drop unique index and modify varchar length in same promise?
+    await knex.schema.alterTable("game_messages", function (table) {
+        table.text("message").notNullable().alter();
+    });
+};
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.down = function (knex) {
+    return knex.schema.table("game_messages", function (table) {
+        table.dropColumn("id");
```

Comment:
When is this column added

Reply:
`table.increments();` creates a auto-increment primary key called 'id'